project/Dependencies.scala

```diff
@@ -18,10 +18,10 @@ object Dependencies {
   val specsVersion = "4.19.2"
   val specsBuild = Seq(
     "specs2-core",
-    "specs2-junit",
-    "specs2-mock"
   ).map("org.specs2" %% _ % specsVersion cross CrossVersion.for3Use2_13)
```

Comment:
Is it possible to remove `CrossVersion.for3Use2_13` now? I think yes or am I wrong?

Reply:
not possible yet 😢 

- https://github.com/xuwei-k/play-ws/commit/7da337ea8706cb9612ae424fb1243a85e105676f
- https://github.com/xuwei-k/play-ws/actions/runs/4642198526/jobs/8215878663#step:9:89

```
[error] -- [E007] Type Mismatch Error: /home/runner/work/play-ws/play-ws/play-ahc-ws-standalone/src/test/scala/play/api/libs/ws/ahc/AhcWSRequestSpec.scala:107:59 
[error] 107 |      newRequest.queryString.get("play") must beSome.which(_.contains("foo1"))
[error]     |                                                           ^
[error]     |Found:    (_$1 : Nothing)
[error]     |Required: ?{ contains: ? }
[error]     |Note that implicit conversions were not tried because the result of an implicit conversion
[error]     |must be more specific than ?{ contains: [applied to ("foo1") returning R] }
[error]     |
[error]     | longer explanation available when compiling with `-explain`
[error] -- [E172] Type Error: /home/runner/work/play-ws/play-ws/play-ahc-ws-standalone/src/test/scala/play/api/libs/ws/ahc/AhcWSRequestSpec.scala:107:78 
[error] 107 |      newRequest.queryString.get("play") must beSome.which(_.contains("foo1"))
[error]     |                                                                              ^
[error]     |Ambiguous given instances: both method booleanAsResult in object AsResult and method errorEffectErrorAsResult in object AsResult match type org.specs2.execute.AsResult[R] of an implicit parameter of method which in class OptionLikeMatcher
[error] -- [E007] Type Mismatch Error: /home/runner/work/play-ws/play-ws/play-ahc-ws-standalone/src/test/scala/play/api/libs/ws/ahc/AhcWSRequestSpec.scala:108:59 
[error] 108 |      newRequest.queryString.get("play") must beSome.which(_.contains("foo2"))
[error]     |                                                           ^
[error]     |Found:    (_$2 : Nothing)
[error]     |Required: ?{ contains: ? }
[error]     |Note that implicit conversions were not tried because the result of an implicit conversion
[error]     |must be more specific than ?{ contains: [applied to ("foo2") returning R] }
[error]     |
[error]     | longer explanation available when compiling with `-explain`
[error] -- [E172] Type Error: /home/runner/work/play-ws/play-ws/play-ahc-ws-standalone/src/test/scala/play/api/libs/ws/ahc/AhcWSRequestSpec.scala:[108](https://github.com/xuwei-k/play-ws/actions/runs/4642198526/jobs/8215878663#step:9:109):78 
[error] 108 |      newRequest.queryString.get("play") must beSome.which(_.contains("foo2"))
[error]     |                                                                              ^
[error]     |Ambiguous given instances: both method booleanAsResult in object AsResult and method errorEffectErrorAsResult in object AsResult match type org.specs2.execute.AsResult[R] of an implicit parameter of method which in class OptionLikeMatcher
[error] -- [E007] Type Mismatch Error: /home/runner/work/play-ws/play-ws/play-ahc-ws-standalone/src/test/scala/play/api/libs/ws/ahc/AhcWSRequestSpec.scala:[109](https://github.com/xuwei-k/play-ws/actions/runs/4642198526/jobs/8215878663#step:9:110):59 
[error] 109 |      newRequest.queryString.get("play") must beSome.which(_.size == 2)
[error]     |                                                           ^
[error]     |Found:    (_$3 : Nothing)
[error]     |Required: ?{ size: ? }
[error]     |Note that implicit conversions were not tried because the result of an implicit conversion
[error]     |must be more specific than ?{ size: ?{ == : [applied to (2) returning R] } }
```

Reply:
@xuwei-k See #755